pkg/operator/ceph/object/health.go

```diff
@@ -119,32 +121,14 @@ func (c *bucketChecker) checkObjectStoreHealth() error {
 
 		Always keep the bucket and the user for the health check, just do PUT and GET because bucket creation is expensive
 	*/
-	var s3AccessKey string
-	var s3SecretKey string
 	var tlsCert []byte
+	var err error
 	s3endpoint := buildDNSEndpoint(BuildDomainName(c.objContext.Name, c.namespacedName.Namespace), c.port, c.objectStoreSpec.IsTLSEnabled())
 
 	// Generate unique user and bucket name
 	bucketName := genUniqueBucketName(c.objContext.UID)
 	userConfig := c.genUserConfig()
 
-	// Create S3 user
-	logger.Debugf("creating s3 user object %q for object store %q", userConfig.UserID, c.namespacedName.Name)
```

Comment:
@leseb: Is there any specific reason to replace `admin ops user` with `health checker`. Both serve different use cases right. IMO it will help to keep them separate if we want to debug some issues with respect to each feature

Reply:
Well, the goal the healtchecher is to ensure the object store works. So I don't see any problem with using the admin ops user to validate this. Why is it a concern?

Reply:
@leseb: It is purely from debugging point of view.
`healthchecker` creates buckets and put data into a bucket frequently.  But `admin ops` user will handle requests for OBC, cephobjectstoreuser etc.

Reply:
Hum so you want to create a checker user with the admin API and then use it?

Reply:
Done!